docs/pages/development/installation.md

```diff
@@ -154,17 +132,17 @@ import App from "./App";
 
 > Note: This will only affect the application in which you make this change. If you want to load multiple projects from a single development app, you'll need to add this import statement to each project.
 
-### Loading published updates
+### Disable packager autostart when building for iOS
 
-Development builds can also open and preview published updates to your app. To add this feature, add `expo-updates@0.8.0` or newer to your app if it isn't already installed, and add a small additional integration in your **AppDelegate.m** and **MainApplication.java** files.
+When you start your project on iOS, the metro bundler will be started automatically. This behavior might not be ideal when you want to use `expo start`. Our recommended solution is to remove the `Start Packager` action from building scripts. To do that you need to open the Xcode, go to "Build Phases" and remove the "Start Packager" action.
 
-1. [Install `expo-modules` in your project](../bare/installing-expo-modules.md).
-2. [Install `expo-updates` in your project](../bare/installing-updates.md).
-3. Make the following changes to complete the integration with `expo-updates`:
+<img src="/static/images/client/remove_start_packager.png" style={{maxWidth: "100%" }}/>
+
+### Loading published updates
 
-<ConfigurationDiff source="/static/diffs/client/app-delegate-updates.diff" />
+Development builds can also open and preview published updates to your app. To add this feature to your existing project, [add `expo-updates` to your app](../bare/installing-updates.md), if it isn't already installed. Otherwise, if you are working from a template, you won't need to make any changes beyond the following commands:
```

Comment:
```suggestion
Development builds can also open and preview published updates to your app. To add this feature to your existing project, [add `expo-updates` to your app](/bare/installing-updates.md), if it isn't already installed. Otherwise, if you are working from a template, you won't need to make any changes beyond the following commands:
```

We should avoid using relative links if possible.

Reply:
While following the guide, I don't understand (from a the first time user's perspective) what are "published updates"? Can we add a little bit more context here or provide a link from an internal docs like: https://docs.expo.dev/workflow/glossary-of-terms/#updates?

I think to be grammatically correct, it could be `publish updates`.

I also think we'd remove the phrase: "if it isn't already installed". I think from the following the guide, we know expo-update pkg isn't installed by default when I create a bare react native app (using CRNA).